File: Assignments/Week_01/quadraticRoots.js
const quadraticRoots = (a,b,c) => {
	let root_part =  (b*b + 4*a*c)**0.5
	let denom = (2*a)

	let root1 = (-b + root_part)/denom
	let root2 = (-b - root_part)/denom

	console.log(root1,root2)
	
}

quadraticRoots(2,2,4)

File: Assignments/Week_01/quadraticRoots.py
def quadraticRoots (a,b,c):
	root_part = (b*b - 4*a*c)**0.5
	denominator = 2*a
	root_one = (-b + root_part)/denominator
	root_two = (-b - root_part)/denominator

	return(root_one, root_two)

print(quadraticRoots(1,1,1))